bors.toml

```
@@ -0,0 +1 @@
+status = ["ci"]
```

I prefer to add the `use-squash-merge` option which makes the commit history look like it was done by normal people instead of the bors bot. See the [Skip'n Go commit history](https://github.com/katharostech/skipngo/commits/master) for an example of commits merged with that option.

```suggestion
status = ["ci"]
use_squash_merge = true
```

---

Squash merges are going to void the advantages of granular commits.

Granular commits allow a more precise bisecting to find bugs, and make understanding steps to implement functionalities easier.

It already happened that I wasn't able to fully understand how to implement a functionality because when I looked at the commit, it was a wall of code.

AFAIK Github preserves the individual commits, but not in the local repository. This means that local tools (IDEs/CLI) won't be able to dig in the history.

---

Valid point. Go ahead and leave it out then.